database/data_generation/data_generation.js

```diff
@@ -0,0 +1,123 @@
+const faker = require('faker');
+const momentRandom = require('moment-random');
+// const pg = require('pg');
+const knex = require('knex')({
+  client: 'pg',
+  connection: 'postgres://postgres:postgres@localhost:5432/analytics',
+});
+
+// const conString = 'postgres://postgres:postgres@localhost:5432/analytics';
+// const client = new pg.Client(conString);
+// client.connect();
+
+const type = ['Apartment', 'Condominium', 'Guesthouse', 'House'];
+const city = ['San Francisco', 'New York', 'Los Angeles', 'Lisbon', 'Barcelona',
+              'London', 'Shanghai', 'Portland', 'Seattle', 'Seoul', 'Bangkok', 'Paris',
+              'Berlin', 'Moscow', 'Dublin', 'Dubai', 'Mexico City', 'Peru', 'Miami',
+              'Rome', 'Antalya', 'Kuala Lumpur', 'Istanbul', 'Hong Kong', 'Singapore'];
+const generateListingHost = view => (
+  {
+    host_id: view.hostId,
+    superhost: view.superhostStatus,
+  }
+);
+
+const generateListings = view => (
+  {
+    listing_id: view.listingId,
+    city: view.city,
+    rating: view.rating,
+    price: view.price,
+    accomodation_type: view.accomodationType,
+    beds: view.beds,
+  }
+);
+
+const bookingBias = (view) => {
+  let bookingCount = Math.floor(Math.random() * 1000);
+  if (view.superhostStatus) {
+    bookingCount += 150;
+  }
+  return bookingCount;
+};
+
+const addView = view => (
+  {
+    date: view.createdAt,
+    listing_id: view.listingId,
+    host_id: view.hostId,
+    viewings: Math.floor(Math.random() * 100000),
+    bookings: bookingBias(view),
+  }
+);
+
+
+const View = num => (
+  {
+    createdAt: momentRandom('2017-12-13', '2017-12-03').format('l'),
```

Comment:
to me, it looks like the start date is later than the end date in `momentRandom()`. Just want to clarify that's how its supposed to be. We're also all using a different date format than `moment.format('l')`

Reply:
This date is just meant for data generating purposes, when I receive data from the client server, all the dates in my database will be the format from Jen